### PR TITLE
palindrome_number function

### PR DESCRIPTION
**Pull request by nikkitorab**
test_returns_true_for_99 and test_returns_true_for_151 failed

### CODE REVIEW

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+  temp = x
+  reverse = 0
+  while(x>0):
+    dig = x%10
```
Comment (sarah-kw) [inline functions/palindrome_number.py]: dig could be more descriptive
```suggestion
    digit = x%10
```

File: functions/palindrome_number.py [hunk 2]
```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+  temp = x
```
Comment (sarah-kw) [inline functions/palindrome_number.py:2]: stylistically (for readability), I would perform the mod manipulations using temp instead of x, and make the final comparison x == reverse

File: tests/palindrome_number_test.py [hunk 3]
```diff
@@ -1,2 +1,34 @@
 import pytest
 from functions.palindrome_number import palindrome_number
+
```
Comment (sarah-kw) [inline tests/palindrome_number_test.py:3]: consider writing tests for edge cases (single digit number, zero)
